src/components/ContactGroups/SelectBox/Option.jsx

```diff
@@ -11,7 +11,8 @@ const Option = props => {
     editedGroupId,
     setEditedGroupId,
     deleteGroup,
-    renameGroup
+    renameGroup,
+    isMulti
```

Comment:
I think it's better to stay close to the react-select API or our ActionsOption. Why not using `withCheckbox` instead of isMulti?

Reply:
oh yes my bad !